src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/serialization/AsylumCaseCallbackDeserializer.java

```diff
@@ -34,7 +30,6 @@ public AsylumCaseCallbackDeserializer(
             );
 
         } catch (IOException e) {
-            LOG.warn("Could not deserialize callback:\n{}\n{}", e.getMessage(), source);
```

Comment:
Any reason why the logs in exceptions were removed?  I ask because sonar sometimes complains about not logging errors.

Reply:
Yeah Sonar said no to logging and throwing in the same block